src/eva/data/ioda_obs_space.py

```diff
@@ -125,8 +125,10 @@ def execute(self, data_collections):
                     collection_name = dataset['name']
 
                     # Read the group
-                    ds = xr.open_dataset(filename, group=group_name, mask_and_scale=False,
-                                         decode_times=False)
+                    timing.start(f'IodaObsSpace: open_dataset')
```

Comment:
Thoughts on adding the filename to the timer (perhaps a debug option?) in case there are multiple files to see which one is slow to read?

Reply:
I've added the filename. It needed a bit of work to allow for longer strings without ruining the look of the output but I think it's OK. Here's an example of what it looks like now:

![Screen Shot 2022-09-23 at 1 54 57 PM](https://user-images.githubusercontent.com/27729500/192028222-80bb5688-c2e3-4831-b32d-02fc03657595.png)